app/js/lib/utils.js

```diff
@@ -0,0 +1,19 @@
+var fs = require('fs');
+var path = require('path');
+
+
+function setPath(pathname, baseDir) {
```

Comment:
Since this is a utility method could you provide a very brief function doc? Just in case others want to use it.
Sorry for being a stickler!
Thanks